### PR TITLE
update ts/ecma

### PR DESCRIPTION
**Pull request by amaanq**

### CODE REVIEW

File: queries/typescript/highlights.scm [hunk 1]
```diff
@@ -52,6 +59,9 @@
 (type_parameters
   ["<" ">"] @punctuation.bracket)
 
+(object_type
+  ["{|" "|}"] @punctuation.bracket)
```
Comment (lucario387) [inline queries/typescript/highlights.scm:63]: This looking wrong, I don't remember object types having this syntax

Comment (amaanq) [reply queries/typescript/highlights.scm:63]: it's in the grammar

Comment (lucario387) [reply queries/typescript/highlights.scm:63]: Just found it

**Review by lucario387 — APPROVED** [review]

File: queries/typescript/highlights.scm (hunk 1 above)
Comment (zweimach) [reply queries/typescript/highlights.scm:63]: @lucario387 is right about this. TypeScript doesn't have this syntax.
It is a syntax for [Flow](https://flow.org/en/docs/types/objects/#exact-and-inexact-object-types), and Flow (in Atom) uses the TS grammar.

Comment (zweimach) [reply queries/typescript/highlights.scm:63]: Thanks for the PR. I didn't know that I would need these additions. 👍🏽

**Pull request merged**